samples/contrib/versioned-pipeline-ci-samples/kaggle-ci-sample/cloudbuild.yaml

```diff
@@ -0,0 +1,138 @@
+steps:
+  - name: "gcr.io/cloud-builders/docker"
+    args:
+      [
+        "build",
+        "-t",
+        "${_GCR_PATH}/kaggle_download:$COMMIT_SHA",
+        "-t",
+        "${_GCR_PATH}/kaggle_download:latest",
+        "${_CODE_PATH}/download_dataset",
+        "-f",
+        "${_CODE_PATH}/download_dataset/Dockerfile",
+      ]
+    id: "BuildDownloadDataImage"
+
```

Comment:
Do we want to push image after we build it?

Reply:
Yes, the final step will do.